packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -137,6 +139,23 @@ class EksClusterStack extends TestStack {
     // apply a kubernetes manifest
     this.cluster.addManifest('HelloApp', ...hello.resources);
   }
+  private assertManifestWithoutValidation() {
+    // apply a kubernetes manifest
+    new eks.KubernetesManifest(this, 'HelloAppWithoutValidation', {
+      cluster: this.cluster,
+      manifest: [{
+        apiVersion: 'v1',
+        kind: 'Service',
+        metadata: { name: 'hello-kubernetes-without-validation' },
+        spec: {
+          type: 'LoadBalancer',
+          ports: [{ port: 80, targetPort: 8080 }],
+          selector: { app: 'hello-kubernetes-without-validation' },
+        },
```

Comment:
Does the `Service` here serve any purpose? Can this be a simple `ConfigMap`?

Reply:
Yes it can just be a `ConfigMap`. Will make the change